lib/membrane/video_compositor/handler/callback_ctx.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Membrane.VideoCompositor.Handler.CallbackCtx do
+  @moduledoc """
+  Structure representing a common part of the context
+  for all of the callbacks.
+  """
+  alias Membrane.{Pad, Time}
+  alias Membrane.VideoCompositor.Scene
+
+  defmacro __using__(fields) do
```

Comment:
[NIT] This seems more explanatory. `fields` suggests, that those are all struct fields, which isn't the case here.
```suggestion
  defmacro __using__(custom_fields) do
```